Task 8/script.js

```diff
@@ -5,3 +5,13 @@ subtraction(a, b) - priima du skaičius ir grąžina jų skirtumą.
 multiplication(a, b) - priima du skaičius ir grąžina jų daugybos rezultatą;
 division(a, b) - priima du skaičius ir grąžina jų dalybos rezultatą;
 ------------------------------------------------------------------------------------ */
+function Calculator() {}
+
```

Comment:
reiktų kuriant objektą užsidėt initial values, pvz.: 
function Calculator(a, b) {
  this.a = a;
  this.b = b;
}